File: src/main/java/um/haberes/report/model/haberes/core/CursoFusionDto.java
package um.haberes.report.model.haberes.core;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class CursoFusionDto {

    private Long cursoFusionId;

    private Long legajoId;

    private int anho = 0;

    private int mes = 0;

    private Integer facultadId;

    private Integer geograficaId;

    private Integer cargoTipoId;

    private Integer designacionTipoId;

    private byte anual = 0;

    private Integer categoriaId;

}

File: src/main/java/um/haberes/report/kotlin/dto/haberes/core/CursoFusionDto.kt
package um.haberes.report.kotlin.dto.haberes.core

data class CursoFusionDto (
    var cursoFusionId: Long? = null,

    var legajoId: Long? = null,
    var anho: Int = 0,
    var mes: Int = 0,
    var facultadId: Int? = null,
    var geograficaId: Int? = null,
    var cargoTipoId: Int? = null,
    var designacionTipoId: Int? = null,
    var anual: Byte = 0,
    var categoriaId: Int? = null,
){
}